accepted/2.17/enhanced-enums/feature-specification.md

```diff
@@ -43,7 +43,8 @@ The grammar of the `enum` declaration becomes:
   `}'
 
 <enumEntry> ::= <metadata> <identifier> <argumentPart>?
-  | <metadata> <identifier> <typeArguments>? `.' <identifier> <arguments>
+  | <metadata> <identifier> <typeArguments>? `.' (<identifier> | `new')
```

Comment:
Should we have a ``<constructorName> ::= <identifier> | `new'`` production somewhere, so we can reuse it everywhere we refer to the name of a named constructor?

That should avoid forgetting a place if we ever change it again.

Reply:
I added it to Dart.g with a 'TODO', then it'll go into the language specification next.